### PR TITLE
Configure dependabot to only observe version catalog file

### PR DESCRIPTION
**Pull request by wzieba**
### Description

The recent FluxC merge caused Dependabot to create PRs for merged submodules, particularly `login` module.

To reduce unnecessary noise as we'll be moving to version catalogs for `login` module as well, let's configure Dependabot to check for dependencies only in version catalog file. Relevant comment: https://github.com/wordpress-mobile/WordPress-Android/pull/21337#issuecomment-2434543581

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by nbradbury** [comment]
@wzieba I'll leave it to @ParaskP7 to review this as he's more well-versed with dependabot. I do have a couple of open dependabot PRs that don't target the version catalog. I'm assuming I should simply close these?

**Comment by wzieba** [comment]
Gotcha @nbradbury !

>  I do have a couple of open dependabot PRs that don't target the version catalog. I'm assuming I should simply close these?

I think we can leave it as it is for now - after merging this PR, I suppose they'll be automatically closed 👍

**Review by ParaskP7 — APPROVED** [review]
👋 @wzieba !

This change indeed makes sense, LGTM, thanks! 🌟

**Pull request merged**
